fortran/toml-f/Portfile

```diff
@@ -5,18 +5,20 @@ PortGroup           cmake 1.1
 PortGroup           compilers 1.0
 PortGroup           github 1.0
 
-github.setup        toml-f toml-f f4f1cc75971f3505679531123bcdcdfaf89d0a83
-version             0.4.1
-revision            1
+github.setup        toml-f toml-f 0.4.2 v
+revision            0
 categories          fortran devel
 license             {Apache-2 MIT}
 maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
 description         TOML parser implementation for data serialization and deserialization in Fortran
 long_description    {*}${description}
 homepage            https://toml-f.readthedocs.io
-checksums           rmd160  1d8f2e4eec4f3094709731ae1548999d969051d1 \
-                    sha256  c453e090216fa09f9d633fda98d2f829994ae6ac86f41ce9f28a057c1610b6ee \
-                    size    437627
+checksums           rmd160  01ef90cdf64c80512e464ae1ed1c9fc6c38688ba \
+                    sha256  6b49013d3bcd1043494c140d7b2da6b0cedd87648e4fc5179fcfcf41226d3232 \
+                    size    406944
+github.tarball_from releases
```

Comment:
is the specific case a dew lines down with a dependency on git still necessary now?

Reply:
Awesome, thanks for pointing out. Looks like we can drop that, since test-drive is included in subprojects folder.